client/src/components/layout/DestinationShow.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+
+const DestinationShow = (props) => {
+  const [destination, setDestination] = useState({
+    id: "",
+    name: "",
+    type: "",
+    location: "",
+    description: "",
+    website: "",
+  });
```

Comment:
can optionally set initial state for `destination` as an empty object so you don't have to keep updating the list of properties here (for a form we would need state to be explicit)